modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -1246,7 +1246,7 @@ public static CompletableFuture<Void> stopAsync(Supplier<CompletableFuture<Void>
      * @param components Array of ignite components to stop.
      * @return CompletableFuture that will be completed when all components are stopped.
      */
-    public static CompletableFuture<Void> stopAsync(ComponentContext componentContext, IgniteComponent... components) {
+    public static CompletableFuture<Void> stopAsync(ComponentContext componentContext, @Nullable IgniteComponent... components) {
```

Comment:
IMHO nullable is confusing there - it feels like components can be nullable, and I think we don't have to annotate that varargs can be empty